pom.xml

```diff
@@ -108,19 +91,19 @@
       </plugin>
 
       <!-- Skips the GWTTestCases during Junit Testing -->
-      <plugin>
```

Comment:
I've turned testing off as it's not being used. This fires the selenium testing with the gwt-driver and it's not working anymore.